blessings/tests/test_sequences.py

```diff
@@ -115,19 +114,25 @@ def child(kind):
             assert 'warnings should have been emitted.'
         warnings.resetwarnings()
 
-    # any binary terminal should do.
-    child(binary_terminals[random.randrange(len(binary_terminals))])
+    # Although any binary terminal should do, FreeBSD has "termcap entry bugs"
+    # that cause false negatives, because their underlying curses library emits
+    # some kind of "warning" to stderr, which our @as_subprocess decorator
+    # determines to be noteworthy enough to fail the test:
+    #     https://gist.github.com/jquast/7b90af251fe4000baa09
+    #
+    # so we chose only one of beautiful lineage:
+    #    http://terminals.classiccmp.org/wiki/index.php/Tektronix_4207
+    child(kind='tek4207-s')
```

Comment:
I didn't realize we had `random` calls in the tests. That's crazy. So this is an improvement for 2 reasons!